contracts/lib/SignatureVerification.sol

```diff
@@ -72,10 +75,15 @@ contract SignatureVerification is SignatureVerificationErrors, LowLevelHelpers {
 
                 // Place final byte on the stack at v.
                 v := byte(0, mload(add(signature, ThreeWords)))
+
+                // Whether v is 27 or 28.
+                // The magic constant has the 27th and 28th bytes
+                // counting from the most significant byte set to 1.
+                vIsValid := byte(v, ECDSA_twentySeventhAndTwentyEighthBytesSet)
```

Comment:
You may as well just flip this entire thing into `vIsInvalid`, flipping all the bytes, and getting rid of the `iszero` (aka `!`).

At least then the saving is reasonable (including pushes: 6 gas in total vs. 18 gas for `iszero(or(eq(v, 27), eq(v, 28)))`, otherwise this is borderline making it more complex for not much savings.

Reply:
Without this currently it is 9 vs 18 gas, which I'd argue is not much of a saving compared to the complexity.